### PR TITLE
[MSPAINT] Fix scrollbar presence on canvas/view fitting

### PR DESCRIPTION
**Pull request by reactos-contributor**
This is a small fix for a scrollbar related edge case functionality exhibited in mspaint (and in a lesser degree also elsewhere in instances of) scrollbar controlled content. As known, the scrollbars presence is given by the size ratio between the accommodating outer frame and the scroll-controlled contents inside it, and it works for the most part. There is, however, a case where the presence of both toolbars is kept even when the contents would fit the frame (if not for the scrollbars themselves that increase the size of the contents' area). Please take the current commit as a demo fix and a provisional correction for mspaint (as I think that the proper fix should reside in common controls).

JIRA issue: [CORE-15096](https://jira.reactos.org/browse/CORE-15096)

The commit exposes the modes defined by canvas/view ratio and improves the UpdateScrollbox() function by taking into account the need for scrollbars in the edge case of 2nd canvas/view mode.

### CODE REVIEW

**Comment by reactos-contributor** [comment]
Ok, Stanislav. I'll do that tonight.

**Comment by reactos-contributor** [comment]
Thank you for review, Benedikt.
Stanislav, I think I've done the change requested by you but I don't know how to mark them as such in Github.

File: base/applications/mspaint/scrollbox.cpp [hunk 1]
```diff
@@ -9,31 +9,90 @@
...
+                        ::GetSystemMetrics(SM_CYHSCROLL) : 0);
+    si.nPage  = sizeScrollBox.cx;
     scrollboxWindow.SetScrollInfo(SB_HORZ, &si);
-    scrollboxWindow.GetClientRect(&clientRectScrollbox);
-    si.nMax   = clientRectImageArea.bottom + 6 - 1;
-    si.nPage  = clientRectScrollbox.bottom;
+
+    si.nMax   = sizeImageArea.cy +
+                    (sp_mx[vmode][hmode].bHoriz == TRUE ?
+                        ::GetSystemMetrics(SM_CXHSCROLL) : 0);
```
Comment (reactos-contributor) [inline base/applications/mspaint/scrollbox.cpp]: I made a small mistake here. It should have been SM_CXVSCROLL instead of SM_CXHSCROLL.

Comment (binarymaster) [reply base/applications/mspaint/scrollbox.cpp]: You can just make changes, amend it to the existing commit, and do force push.

**Pull request merged**